pkg/api/types.go

```diff
@@ -37,7 +37,7 @@ type WorkflowSpec struct {
 	// More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata
 	k8sApi.ObjectMeta `json:"metadata,omitempty"`
 
-	ActiveDeadlineSeconds *int64 `json:"activeDeadlineSeconds,omitempty"`
+	ActiveDeadlineSeconds int64 `json:"activeDeadlineSeconds,omitempty"`
```

Comment:
Please use English ;)
I agree with you, but maybe we should fist ask Ad what the advantage of using pointers to integers is?